data/MadhoolikaNagesh.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Madhooolika Nagesh",
+  "type": "personal",
+  "bio": "ISE Student **Tech enthusiast**.Second year Engineering Student ",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Madhoolika_N",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Linked In",
+      "url": "www.linkedin.com/in/madhoolika-nagesh",
```

You might want to use `https` instead, otherwise, it could lead to unwanted links.
```suggestion
      "url": "https://www.linkedin.com/in/madhoolika-nagesh",
```